### PR TITLE
[karma] Adopt networking.k8s.io/v1

### LINKED ISSUE #42: [karma] Upgrade Ingress definition as extensions/v1beta1 Ingress is deprecated
When activating the ingress with the karma chart, the following warning is displayed.

Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress

Could it be possible to update the Ingress manifest to use the networking.k8s.io/v1 definition?

I was unable to deploy in on OpenShift 4.7 I had to create one manually and turned Ingress off.

### PR DESCRIPTION
**Pull request by yurrriq**
Fixes #42. Note that this drops support for k8s <1.19.

### CODE REVIEW

**Comment by desaintmartin** [comment]
Thanks! Could you add a test value for the ingress?

**Comment by yurrriq** [comment]
Sure. Just a fake hostname or something?

**Review by desaintmartin — APPROVED** [review]
Thanks!

File: charts/karma/Chart.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+apiVersion: v2
+type: application
 description: A Helm chart for Karma - an UI for Prometheus Alertmanager
 name: karma
 home: https://github.com/prymitive/karma
 sources:
   - https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/karma
   - https://github.com/prymitive/karma
-version: 1.8.3
+version: 2.0.0
```
Comment (desaintmartin) [inline charts/karma/Chart.yaml:9]: Hm, actually, one last thing, could you add a minimum kubernetes version here?

Comment (yurrriq) [reply charts/karma/Chart.yaml:9]: Sure! Is `>= 1.19-0` ok?

Comment (desaintmartin) [reply charts/karma/Chart.yaml:9]: Yes!

Comment (yurrriq) [reply charts/karma/Chart.yaml:9]: Done, I forgot to say.

**Comment by desaintmartin** [comment]
Waiting for https://github.com/wiremind/wiremind-helm-charts/pull/56 to be merged.

**Comment by yurrriq** [comment]
Sorry I missed this... rebased now :crossed_fingers:

**Pull request merged**